app/controllers/application_controller.rb

```diff
@@ -72,7 +72,8 @@ def self.disable_site_header!(*args)
 
   private
   def storable_location?
-    request.get? && is_navigational_format? && !devise_controller? && !request.xhr?
+    request.get? && is_navigational_format? && !devise_controller? && !request.xhr? &&
+      request.fullpath != '/site.webmanifest'
```

Comment:
For now, I've just kept it simple with just the `request.fullpath != '/site.webmanifest'` condition. If we want to add more urls later, we can refactor.